yw_website/script_upload/migrations/0001_initial.py

```diff
@@ -15,9 +15,21 @@ class Migration(migrations.Migration):
             name='Document',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(blank=True, max_length=255)),
+                ('workflow', models.ImageField(default='workflow.png', upload_to='document/')),
                 ('description', models.CharField(blank=True, max_length=255)),
                 ('document', models.FileField(upload_to='documents/')),
                 ('uploaded_at', models.DateTimeField(auto_now_add=True)),
             ],
         ),
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('first_name', models.CharField(max_length=255)),
+                ('last_name', models.CharField(max_length=255)),
```

Comment:
I thought we werent tracking first and last name in User?

Reply:
This is from brooke's code. We haven't decided what we want to do with user. Also you are looking at an auto-generated binary not any actual code.

Reply:
all that is needed to create an account is a username and a password right?

Reply:
Not sure. I think so